certificate/views.py

```diff
@@ -2437,26 +2438,34 @@ def fossee_internship_cerificate_download(request):
     return render_to_response('fossee_internship_cerificate_download.html', context, ci)
 
 
-def create_fossee_internship_cerificate(certificate_path, name, qrcode, type, paper, internship_project_duration, student_edu_detail, student_institute_detail, superviser_name_detail, workshop, file_name):
+def create_fossee_internship_cerificate(
+      certificate_path, name, qrcode,
+      wtype, paper, internship_project_duration, student_edu_detail,
+      student_institute_detail, superviser_name_detail,
+      workshop, file_name):
     error = False
     try:
         download_file_name = None
-        year = internship_project_duration[internship_project_duration.find('to')-5:internship_project_duration.find('to')].strip()
-        if type == 'P':
+        year = internship_project_duration[
```

Comment:
I hope this is fine and will work in all the cases!
For now I am ignoring this but see to it that it's generic and implemented in a more better way.
If that is the case(if change needed) then only create an issue for the same.

Reply:
ok will create a issue